README.md

```diff
@@ -5,7 +5,7 @@
 > Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: Make simple calculations. Read a random math-related quote.
 
 ![screenshot](./public/app1.png)
-
+![test's screenshot](./public/test.png)
```

Comment:
- [ ] Please remove the default test that comes with CRA when the app was first created, as it's currently failing.

![image](https://user-images.githubusercontent.com/79114233/149179401-7fe77655-ca81-47f2-b0c3-c620c7a6792d.png)

Reply:
- [ ] Please write tests that simulate user interaction as required by Microverse.

> - Create unit tests for all React components:
>   - Use Jest snapshots to test the components.
>   - Use React Testing Library to simulate user interaction.

This should simulate clicks on links, clicks on the calculator, and expect the React Tree to rerender properly every time. These are especially useful as they allow to test the integration of the logic and the UI much quicker than testing the app manually each time. [Read more about it here.](https://testing-library.com/docs/react-testing-library/example-intro)